docs/build/build-process-management/build-manually-or-with-triggers.md

```diff
@@ -131,6 +131,32 @@ If your commit message includes `[skip ci]` or `[ci skip]`, your workflow will b
 
 If your merge request comment includes `[retry]`, your workflow will be retried.
 
+### Auto Cancel Redundant Pipeline
+
+<Screenshot url='https://cdn.appcircle.io/docs/assets/auto-cancel-redundant-pipeline.png' />
+
+You can enable automatic cancellation of redundant builds when a new workflow run is triggered with the same configuration, workflow, branch, and trigger type. This helps avoid waiting for outdated builds to complete.
+
+To enable this feature:
+
+1. Go to the build profile and select configurations.
+2. Scroll down and activate auto cancel redundant pipeline.
+3. Click save changes.
+
+The Auto-Cancel Redundant Pipeline (ACRP) mechanism works based on four key parameters:
+- Configuration
+- Workflow
+- Branch
+- Trigger type
+
+If a new build is triggered with the same values for all these parameters, any previously queued or running build is automatically cancelled.
```

Comment:
Can you attach a Build Profile image here that was canceled using ACRP? For example, the previous 1 or 2 builds may have been Canceled.